tests/testapp/urls.py

```diff
@@ -9,4 +9,34 @@
     path('base_org_view', views.base_org_view, name='test_base_org_permission_view'),
     path('org_field_view', views.org_field_view, name='test_organization_field_view'),
     path('error_field_view', views.error_field_view, name='test_error_field_view'),
+    path(
+        'books_list_member_view/<str:shelf_id>',
```

Comment:
It's a shame the URLs in this file are not realistic, let's not follow that convention.

Let's try to use realistic, RESTful URLs which will help the readers of the code to understand what these mixins are for.

This is a view which has a parent right? So the URL should be something like:

`/member/shelf/<str:shelf_id>/books/`